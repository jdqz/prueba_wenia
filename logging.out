2023-06-05 13:32:10 INFO     API queries_quota: 60
2023-06-05 13:51:28 INFO     API queries_quota: 60
2023-06-05 13:52:15 INFO     API queries_quota: 60
2023-06-05 13:52:54 INFO     API queries_quota: 60
2023-06-05 13:54:00 INFO     API queries_quota: 60
2023-06-05 13:54:13 INFO     API queries_quota: 60
2023-06-05 13:54:25 INFO     API queries_quota: 60
2023-06-05 13:55:44 INFO     API queries_quota: 60
2023-06-05 13:56:41 INFO     API queries_quota: 60
2023-06-05 13:57:33 INFO     API queries_quota: 60
2023-06-05 14:10:05 INFO     API queries_quota: 60
2023-06-05 14:11:02 INFO     API queries_quota: 60
2023-06-05 14:11:16 INFO     API queries_quota: 60
2023-06-05 14:19:54 INFO     API queries_quota: 60
2023-06-05 14:20:51 INFO     API queries_quota: 60
2023-06-05 15:38:13 INFO     API queries_quota: 60
