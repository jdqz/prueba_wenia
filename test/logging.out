2023-06-05 16:16:13 INFO     API queries_quota: 60
2023-06-05 16:17:11 INFO     API queries_quota: 60
